src/Runner.Worker/Handlers/ContainerActionHandler.cs

```diff
@@ -68,6 +67,10 @@ public async Task RunAsync(ActionRunStage stage)
 
                 Data.Image = imageName;
             }
+            else
```

Comment:
Currently, we allow local (local as in referenced by path in `uses: ./action-path`) container actions to specify images like so: `image: ubuntu:latest`. This goes against the documentation, so we're fixing that here. 

We will now throw on local container actions that specify the image in a format like `image: ubuntu:latest`, because now we require `image: docker://ubuntu:latest` strictly, in accordance with the [documentation](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-using-public-docker-registry-container).